src/components/StarterPack/StarterPackCard.tsx

```diff
@@ -104,14 +110,42 @@ export function Link({
   }
 
   return (
-    <InternalLink
```

Comment:
I think we actually want to keep this a link. E.g. cmd+click should open in new tab. Any reason for this change?